temba/channels/views.py

```diff
@@ -1539,13 +1539,13 @@ class VumiClaimForm(forms.Form):
                                           help_text=_("Your Vumi account key as found under Account -> Details"))
             conversation_key = forms.CharField(label=_("Conversation Key"),
                                                help_text=_("The key for your Vumi conversation, can be found in the URL"))
-            transport_name = forms.CharField(label=_("Transport Name"), required=False,
-                                             help_text=_("The name of the Vumi transport you will use to send and receive messages"))
+            api_url = forms.URLField(label=_("API URL"), required=False,
+                                     help_text=_("Customise VUMI API URL. Defaults to: 'https://go.vumi.org/api/v1/go/http_api_nostream'. Use the URL without trailing slash. '/<conversation_key>/messages.json' will be automatically attached to the end of the specified URL."))
```

Comment:
"Custom VUMI API Endpoint. Leave blank to use default of: https://go.vumi.org/api/v1/go/http_api_nostream"